lib/main_common.pm

```diff
@@ -630,6 +630,16 @@ sub we_is_applicable {
       && get_var('MIGRATION_REMOVE_ADDONS', '') !~ /we/;
 }
 
+sub libreoffice_is_applicable {
+    # for opensuse libreoffice package has ExclusiveArch:  aarch64 %{ix86} x86_64
+    # do not know for SLE (so assume built for all)
+    return if (!is_opensuse);
```

Comment:
in my understanding for sle it will return undef so libreoffice will not run for SLE at all . is it what you expected ?

Reply:
We have sled, and I think there we test LibreOffice

Reply:
mea culpa, I am updating to return 1 here.